pkg/vmprovider/providers/vsphere2/session/session_vm_update.go

```diff
@@ -681,14 +681,9 @@ func (s *Session) customize(
 	vmCtx context.VirtualMachineContextA2,
 	resVM *res.VirtualMachine,
 	cfg *vimTypes.VirtualMachineConfigInfo,
-	updateArgs *VMUpdateArgs) error {
-
-	err := vmlifecycle.DoBootstrap(vmCtx, resVM.VcVM(), cfg, s.K8sClient, updateArgs.NetworkResults, updateArgs.BootstrapData)
-	if err != nil {
-		return err
-	}
+	bootstrapArgs vmlifecycle.BootstrapArgs) error {
 
-	return nil
+	return vmlifecycle.DoBootstrap(vmCtx, resVM.VcVM(), cfg, bootstrapArgs)
```

Comment:
The `session.customize()` method doesn't seem to require anything from `session`, as it simply calls another function here. Would it be better to remove it and directly invoke `vmlifecycle.DoBootstrap()` from the caller?

Reply:
I thought of that, but I left as-is in case we reintroduce other bits to `customize` later. Especially given how much this in the critical path, it also makes rebases easier for folks working on branches in the same area.